templates/english/basic-page-template-table-of-contents.html

```diff
@@ -0,0 +1,159 @@
+<!-- TO DO: Remove all comments before deploying your code to production. -->
```

Comment:
Could we change all instances of `TO DO` to `TODO`? I believe the standard usage is the first one, and I know IDEs also grab `TODO` and create a nice list out of it
```suggestion
<!-- TODO: Remove all comments before deploying your code to production. -->
```

Reply:
All updated @daine :)